lochness/rpms/__init__.py

```diff
@@ -227,13 +228,16 @@ def sync(Lochness, subject, dry=False):
         else:
             latest_pull_mtime = 0
 
+        if len(source_df) == 0:  # do not save if the dataframe is empty
+            continue
+
         # if last_modified date > latest_pull_mtime, pull the data
         source_df['LastModifiedDate'] = pd.to_datetime(
                 source_df['LastModifiedDate'])
         if not source_df['LastModifiedDate'].max() > \
                 pd.to_datetime(latest_pull_mtime):
```

Comment:
I am seeing this just now. Why don't we construct it as:
```python
if source_df['LastModifiedDate'].max() <= pd.to_datetime(latest_pull_mtime):
```

Reply:
looks good

Reply:
8ffdc3c